cluster/gce/gci/configure-helper.sh

```diff
@@ -1796,20 +1748,9 @@ function start-kube-addons {
   if [[ "${NETWORK_POLICY_PROVIDER:-}" == "calico" ]]; then
     setup-addon-manifests "addons" "calico-policy-controller"
```

Comment:
should ENABLE_IP_MASQ_AGENT=true be added where network policy provider is set?

Reply:
we are thinking of doing that in a separate PR later, since it might need more changes for example node labels, etc.

Reply:
ok, I think I was confusing that with "ipMasq": true